<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stanford Artificial Retina Project | Retina Chip v1.0 Experimental Visualization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="7,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_top" stretch="1,1">
        <item>
         <widget class="PlotWidget" name="arrayMap" native="true"/>
        </item>
        <item>
         <layout class="QGridLayout" name="channelTracesGrid"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_bottom">
        <item>
         <widget class="PlotWidget" name="miniMap" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="PlotWidget" name="noiseHistogram" native="true"/>
          </item>
          <item>
           <widget class="PlotWidget" name="spikeRatePlot" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1272</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpdateSession"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionAnalysisParameters"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionIndividualChannelInfo"/>
    <addaction name="actionListElectrodesInfo"/>
    <addaction name="separator"/>
    <addaction name="actionGUIProfiler"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionUpdateSession">
   <property name="text">
    <string>New session...</string>
   </property>
  </action>
  <action name="actionLoad_real_time_stream">
   <property name="text">
    <string>Load real-time stream...</string>
   </property>
  </action>
  <action name="action_changeLayout">
   <property name="text">
    <string>Change visualization style...</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionParameter_1">
   <property name="text">
    <string>Parameter 1</string>
   </property>
  </action>
  <action name="actionParameter_2">
   <property name="text">
    <string>Parameter 2</string>
   </property>
  </action>
  <action name="actionParameter_3">
   <property name="text">
    <string>Parameter 3</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Session parameters...</string>
   </property>
  </action>
  <action name="actionAnalysisParameters">
   <property name="text">
    <string>GUI preferences...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExport_to">
   <property name="text">
    <string>Export to...</string>
   </property>
  </action>
  <action name="actionParameter_4">
   <property name="text">
    <string>Toggle highlight spikes</string>
   </property>
  </action>
  <action name="actionParameter_5">
   <property name="text">
    <string>Toggle auto axis</string>
   </property>
  </action>
  <action name="actionParameter_6">
   <property name="text">
    <string>Parameter 1</string>
   </property>
  </action>
  <action name="actionParameter_7">
   <property name="text">
    <string>Parameter 2</string>
   </property>
  </action>
  <action name="actionParameter">
   <property name="text">
    <string>Parameter 3</string>
   </property>
  </action>
  <action name="actionParameter_8">
   <property name="text">
    <string>Parameter 1</string>
   </property>
  </action>
  <action name="actionParameter_9">
   <property name="text">
    <string>Parameter 2</string>
   </property>
  </action>
  <action name="actionParameter_10">
   <property name="text">
    <string>Parameter 3</string>
   </property>
  </action>
  <action name="action_tl_arraymap">
   <property name="text">
    <string>Array Map</string>
   </property>
  </action>
  <action name="action_tr_arraymap">
   <property name="text">
    <string>Array Map</string>
   </property>
  </action>
  <action name="action_bl_arraymap">
   <property name="text">
    <string>Array Map</string>
   </property>
  </action>
  <action name="action_br_arraymap_2">
   <property name="text">
    <string>Array Map</string>
   </property>
  </action>
  <action name="action_tl_channels">
   <property name="text">
    <string>Channel Traces</string>
   </property>
  </action>
  <action name="action_bl_channels">
   <property name="text">
    <string>Channel Traces</string>
   </property>
  </action>
  <action name="action_br_channels_2">
   <property name="text">
    <string>Channel Traces</string>
   </property>
  </action>
  <action name="action_tl_spikerate">
   <property name="text">
    <string>Spike Rate Plot</string>
   </property>
  </action>
  <action name="action_tl_electrodeminimap">
   <property name="text">
    <string>Electrode Minimap</string>
   </property>
  </action>
  <action name="action_hierlemann">
   <property name="text">
    <string>Hierlemann</string>
   </property>
  </action>
  <action name="action_modifiedhierlemann">
   <property name="text">
    <string>Modified Hierlemann</string>
   </property>
  </action>
  <action name="action_Highpass">
   <property name="text">
    <string>Highpass</string>
   </property>
  </action>
  <action name="actionSpike_detection">
   <property name="text">
    <string>Spike detection...</string>
   </property>
  </action>
  <action name="action_npz">
   <property name="text">
    <string>Numpy file (.npz)</string>
   </property>
  </action>
  <action name="action_mat">
   <property name="text">
    <string>MATLAB file (.mat)</string>
   </property>
  </action>
  <action name="action_save_npz">
   <property name="text">
    <string>Numpy file (.npz)</string>
   </property>
  </action>
  <action name="action_save_mat">
   <property name="text">
    <string>MATLAB file (.mat)</string>
   </property>
  </action>
  <action name="action_tr_channels">
   <property name="text">
    <string>Channel Traces</string>
   </property>
  </action>
  <action name="action_tr_spikerate">
   <property name="text">
    <string>Spike Rate Plot</string>
   </property>
  </action>
  <action name="action_tr_electrodeminimap">
   <property name="text">
    <string>Electrode Minimap</string>
   </property>
  </action>
  <action name="action_bl_spikerate">
   <property name="text">
    <string>Spike Rate Plot</string>
   </property>
  </action>
  <action name="action_bl_electrodeminimap">
   <property name="text">
    <string>Electrode Minimap</string>
   </property>
  </action>
  <action name="action_br_spikerate_2">
   <property name="text">
    <string>Spike Rate Plot</string>
   </property>
  </action>
  <action name="action_br_electrodeminimap_2">
   <property name="text">
    <string>Electrode Minimap</string>
   </property>
  </action>
  <action name="action_save_csv">
   <property name="text">
    <string>CSV file (.csv)</string>
   </property>
  </action>
  <action name="action_csv">
   <property name="text">
    <string>CSV file (.csv)</string>
   </property>
  </action>
  <action name="actionEntire_window">
   <property name="text">
    <string>Entire window</string>
   </property>
  </action>
  <action name="actionWindow_1">
   <property name="text">
    <string>Window 1</string>
   </property>
  </action>
  <action name="actionWindow_2">
   <property name="text">
    <string>Window 2</string>
   </property>
  </action>
  <action name="actionWindow_3">
   <property name="text">
    <string>Window 3</string>
   </property>
  </action>
  <action name="actionWindow_4">
   <property name="text">
    <string>Window 4</string>
   </property>
  </action>
  <action name="action_tl_tracesearch">
   <property name="text">
    <string>Trace Search</string>
   </property>
  </action>
  <action name="action_tr_tracesearch">
   <property name="text">
    <string>Trace Search</string>
   </property>
  </action>
  <action name="action_bl_tracesearch">
   <property name="text">
    <string>Trace Search</string>
   </property>
  </action>
  <action name="action_br_tracesearch_2">
   <property name="text">
    <string>Trace Search</string>
   </property>
  </action>
  <action name="actionToggle_light_dark_mode">
   <property name="text">
    <string>Toggle light/dark mode</string>
   </property>
  </action>
  <action name="actionToggle_profiling">
   <property name="text">
    <string>Toggle profiling</string>
   </property>
  </action>
  <action name="action_H0Bandpass">
   <property name="text">
    <string>h0 Bandpass</string>
   </property>
  </action>
  <action name="action_Auto">
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="action_FastBandpass">
   <property name="text">
    <string>Fast Bandpass</string>
   </property>
  </action>
  <action name="action_FasterBandpass">
   <property name="text">
    <string>Faster Bandpass</string>
   </property>
  </action>
  <action name="action_Litke">
   <property name="text">
    <string>Litke</string>
   </property>
  </action>
  <action name="action_None">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionSpike_threshold">
   <property name="text">
    <string>Spike threshold...</string>
   </property>
  </action>
  <action name="actionTime_window">
   <property name="text">
    <string>Time window...</string>
   </property>
  </action>
  <action name="actionIndividualChannelInfo">
   <property name="text">
    <string>Individual channel info...</string>
   </property>
  </action>
  <action name="actionChannelAmplitudeHistogram">
   <property name="text">
    <string>Channel amplitude histogram...</string>
   </property>
  </action>
  <action name="actionToggleDarkMode">
   <property name="text">
    <string>Toggle dark mode</string>
   </property>
  </action>
  <action name="actionToggleProfiling">
   <property name="text">
    <string>Toggle profiling</string>
   </property>
  </action>
  <action name="actionListElectrodesInfo">
   <property name="text">
    <string>List of electrodes info...</string>
   </property>
  </action>
  <action name="actionGUIProfiler">
   <property name="text">
    <string>GUI Profiler...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>